content/docs/command-reference/params/index.md

```diff
@@ -144,6 +147,51 @@ $ dvc run -n train -d logs/ -o users.csv \
           python train.py
 ```
 
+## Examples: Print all parameters
+
+Following the previous example, we can use `dvc params diff` to list all of the
+param values available in the <abbr>workspace</abbr>:
+
+```dvc
+$ dvc params diff
+Path         Param           Old    New
+params.yaml  lr              —      0.0041
+params.yaml  process.bow     —      15000
+params.yaml  process.thresh  —      0.98
+params.yaml  train.epochs    —      70
+params.yaml  train.layers    —      9
+```
+
+This command shows the difference in parameters between the workspace and the
+last committed version of the `params.yaml` file. In our example there's no
+previous version, which is why all `Old` values are `—`.
+
+## Example: Parameterize stages in `dvc.yaml`
+
+Let's say we have a `params.yaml` file with the following contents:
+
+```yaml
+models:
+  us:
+    thresh: 10
+    filename: 'model-us.hdf5'
+```
+
+To use those values in `dvc.yaml` itself, you can use the special `${}`
+expression:
+
+```yaml
+stages:
+  build-us:
+    cmd: python script.py --out ${models.us.filename}
+    outs:
+      - ${models.us.filename}
+```
+
+> Note that nothing prevent you from using the same parameters in your stages
```

Comment:
prevents

Reply:
not sure I understand why do we need this note tbh

Reply:
a better question would be - if I don't specify it in `params` and still use in `dvc.yaml` does it affect anything at all?  E.g. what does `dvc status` show if I change `models.us.filename` in the params file?

Reply:
> why do we need this note

We don't need it. I guess I forgot to delete it from this example after we discussed that topic. Removed

> if I don't specify it in params and still use in dvc.yaml does it affect anything

`dvc params diff` shows everything in the params file whether you use it in dvc.yaml in any way or not. It's not directly related to 2.0 dvc.yaml parameterization but we can review in the misc updates PR... Will mention you there if I change anything.

Reply:
> if I don't specify it in params and still use in dvc.yaml

If you meant values from `vars` then yes there are changes to how params diff works but I haven't finished all those notes in this PR. It's the last open check box in the description of this PR.